src/package/Microsoft.NET.Test.Sdk/Microsoft.NET.Test.Sdk.csproj

```diff
@@ -0,0 +1,48 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">..\..\..\</TestPlatformRoot>
```

Comment:
Does it need to be set here?

Reply:
We only need it for importing `TestPlatform.Settings.targets` for now, but I decided to have it like this to be in line with other projects that do this.